README.md

```diff
@@ -10,6 +10,12 @@ The benchmark suite should be self contained in terms of dependencies,
 except for the torch products which are intended to be installed separately so
 different torch versions can be benchmarked.
 
+### Required system packages
+On Ubuntu, the installation requires python3-dev and build-essential packages preinstalled.
+```
+sudo apt-get install build-essential python3-dev
```

Comment:
It is bettert to list all packages needed here: https://github.com/pytorch/benchmark/blob/main/docker/gcp-a100-runner-dind.dockerfile#L10